components/clarinet-deployments/src/lib.rs

```diff
@@ -350,14 +345,10 @@ pub async fn generate_default_deployment(
                     ))
                 }
             };
-            // Download the code
-            let (_source, epoch, clarity_version, _contract_location) =
-                requirements::retrieve_contract(&contract_id, &cache_location, &file_accessor)
```

Comment:
the purpose of this PR is to call `retrieve_contract` for each item in the queue and not just here